docs/csharp/tutorials/intro-to-csharp/numbers-in-csharp.yml

```diff
@@ -293,15 +293,14 @@ items:
     > This online coding experience is in preview mode. If you encounter problems, please report them [on the dotnet/try repo](https://github.com/dotnet/try/issues).
 
 - content: |
-    You've completed the "Numbers in C#" interactie tutorial. 
-    
-    You can continue these tutorials on your own development environment. Learn the basics of [local development](local-environment.md) and then pick a tutorial. You can directly to the next tutorial, or start again at with the [numbers in C#](numbers-in-csharp-local.md) tutorial.
-    
+    You've completed the "Numbers in C#" interactie tutorial. You can click the **Next** link below to start the **Branches and loops interactive tutorial**, or you can visit the [.NET site](https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial) to download the .NET Core SDK, create a project on your machine, and keep coding. The "Keep Learning" step brings you back to these tutorials.
```

Comment:
- interactie --> interactive
- No **Next** link; it reads **Branches and loops in C#**
- "Keep Learning" doesn't appear.